conf/cmi/core.extension.yml

```diff
@@ -114,6 +118,7 @@ module:
   update: 0
   update_helper: 0
   user: 0
+  variationcache: 0
```

Comment:
What is this?

Reply:
It's a requirement for the Group module. https://www.drupal.org/project/variationcache